flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/ChangelogMode.java

```diff
@@ -51,6 +58,14 @@ public static ChangelogMode insertOnly() {
         return INSERT_ONLY;
     }
 
+    /**
+     * Shortcut for an upsert changelog that describes idempotent updates on a key and thus does not
+     * contain {@link RowKind#UPDATE_BEFORE} rows.
+     */
+    public static ChangelogMode upsert() {
```

Comment:
Could you use this method in valus source and sink? This can verify it works for the planner. 

https://github.com/apache/flink/blob/c6147f2df2e81c1b1db6aa43b3a827a8c4148af1/flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java#L1240